pom.xml

```diff
@@ -158,12 +159,7 @@
             <dependency>
                 <groupId>io.opentracing</groupId>
                 <artifactId>opentracing-api</artifactId>
-                <version>0.30.0</version>
-            </dependency>
-            <dependency>
-                <groupId>com.uber.jaeger</groupId>
-                <artifactId>jaeger-core</artifactId>
-                <version>0.21.0</version>
```

Comment:
`dependencyManagement` section defines dependency artifact versions (and exclusions) for the whole hierarchy of the project poms - bring it back (with the new `gson` exclusion, if needed).

Reply:
resolved